### PR TITLE
add allocation delay

### PR DESCRIPTION
**Pull request by gpsanant**
Makes the allocation delay configurable and immutable by fixing git setup on #740

### CODE REVIEW

File: script/deploy/mainnet/Deploy_Strategy_Factory.s.sol [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+     * @notice Deploy StrategyFactory for Mainnet
+     */
+
+    function _deployStrategyFactory() internal {
+        strategyFactoryImplementation = new StrategyFactory(
+            strategyManager
+        );
+
+        
+
```
Comment (0xClandestine) [inline script/deploy/mainnet/Deploy_Strategy_Factory.s.sol:59]: Run `forge fmt src/contracts`

**Review by 0xClandestine — CHANGES_REQUESTED** [review]
lgtm

File: script/deploy/mainnet/Deploy_Strategy_Factory.s.sol (hunk 1 above)
Comment (gpsanant) [reply script/deploy/mainnet/Deploy_Strategy_Factory.s.sol:59]: this is in the script folder?

Comment (0xClandestine) [reply script/deploy/mainnet/Deploy_Strategy_Factory.s.sol:59]: oh rip lol

**Review by 0xClandestine — APPROVED** [review]

**Pull request merged**